### PR TITLE
added NWB export

### PR DESCRIPTION
**Pull request by ttngu207**

### CODE REVIEW

File: element_animal/export/nwb.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    subject_info = subject_query.fetch1()
+
+    return pynwb.file.Subject(
+        subject_id=subject_info.pop('subject'),
+        sex=subject_info.pop('sex'),
+        date_of_birth=datetime.combine(subject_info.pop('subject_birth_date'),
+                                       datetime.strptime('00:00:00', '%H:%M:%S').time()),
+        description=json.dumps(subject_info, default=str),
+        genotype=' x '.join((subject.Line.Allele * subject.Subject.Line & subject_key).fetch('allele')),
+        species='Mus musculus'
```
Comment (CBroz1) [inline element_animal/export/nwb.py:22]: Is it safe to hardcode pieces in the export function? My instinct is to instead set as a default within the Line or Subject class, including a note that NWB prefers latin names.

Comment (CBroz1) [reply element_animal/export/nwb.py:22]: PR to [ttngu207/element-animal](https://github.com/ttngu207/element-animal) will resolve this

**Comment by kabilar** [comment]
Hi @ttngu207, would it be okay to close this pull request as your changes are now in #16?

**Comment by ttngu207** [comment]
@kabilar , yes, let's close this one and work on the #16

File: element_animal/export/nwb.py (hunk 1 above)
Comment (bendichter) [reply element_animal/export/nwb.py:22]: where are we on this?

**Pull request closed without merging**